transforms/angle-brackets/index.js

```diff
@@ -26,9 +28,40 @@ function getOptions() {
   return options;
 }
 
+function helperNames(helperPaths) {
+  return [
+    ...new Set(
+      helperPaths
+        .filter(name => name.includes('/helpers/') || name.includes('/helper'))
+        .filter(name => !name.includes('/-'))
+        .map(name => {
+          let path = name.split('/helpers/');
+          return path.pop();
+        })
+        .filter(name => !name.includes('/'))
+    ),
+  ];
+}
+
+function getHelperData(telemetry) {
```

Comment:
If you export this, you can write some unit tests. I would personally love to see some tests here...

Reply:
Yep, I definitely want to write some tests